torch/testing/_internal/common_utils.py

```diff
@@ -1673,6 +1673,20 @@ def check_if_enable(test: unittest.TestCase):
                         "rocm": TEST_WITH_ROCM,
                         "asan": TEST_WITH_ASAN
                     }
+
+                    invalid_platforms = list(filter(lambda p: p not in platform_to_conditional, platforms))
+                    if len(invalid_platforms) > 0:
+                        invalid_plats_str = ", ".join(invalid_platforms)
+                        valid_plats = ", ".join(platform_to_conditional.keys())
+
+                        print(f"Test {disabled_test} is disabled for some unrecognized ",
```

Comment:
Sorry for the late review, but wouldn't this be super noisy (as it will be printed once per test, wouldn't it?)
I.e. how about using `warnings.warn` instead?

Reply:
Unless I'm misreading the code, this would be only printed for the specific test that's been disabled (not once for every test if that was your concern)

I did consider using `warnings.warn` but I was concerned that we might promote warnings to errors and cause CI to fail.  We can modify this though if that's not the case.